backend/notifications/models.py

```diff
@@ -8,6 +8,7 @@ class NotificationType(enum.Enum):
     NEW_COMMENT_TO_COMMENT = 2
     NEW_ANSWER_TO_ANSWER = 3
     NEW_COMMENT_TO_DOCUMENT = 4
+    UPDATE_TO_TESTIMONIAL_APPROVAL_STATUS = 5
```

Comment:
I think it's worth creating a custom DB migration to enable this setting for everyone by default. In `auth_backend.py` I see you've edited the default setting for new users, but existing users (which there are over a thousand on BI) will not default to receiving notifications.

The migration should create a `NotificationSetting` with `UPDATE_TO_TESTIMONIAL_APPROVAL_STATUS` for every existing user.

It might be worth having a reverse migration too, but I'm not sure what the correct semantics should be. Maybe delete every row that has `UPDATE_TO_TESTIMONIAL_APPROVAL_STATUS`?